packages/react-test-renderer/package.json

```diff
@@ -17,16 +13,11 @@
   "dependencies": {
     "fbjs": "^0.8.16",
     "object-assign": "^4.1.1",
-    "prop-types": "^15.6.0"
+    "prop-types": "^15.6.0",
+    "react-is": "^16.3.0-alpha.2"
```

Comment:
Will this require updates to our release/build script to keep these deps in sync? Needs follow up.

Reply:
hmm i don't know, I was conflating workspaces and lerna, the latter of which handles the updates

Reply:
These versions are bumped by our release script manually. It's fine. I'll add it to my list of follow up things.

Reply:
Addressed via 610df95